document_page_environmental_aspect/README.rst

```diff
@@ -0,0 +1,73 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+Document Management - Wiki - Environmental Aspects
+==================================================
+
+This module was written to extend the functionality of Environment Management System. Add a document template for Environmental Aspects.
```

Comment:
This module extends the functionality of Environment Management System and add a document template